includes/output.inc

```diff
@@ -144,6 +144,11 @@ function drush_format($input, $metadata = NULL, $format = NULL) {
  */
 function dt($string, $args = array()) {
   $output = NULL;
+
+  if (!is_array($args)) {
+    $args = array($args);
```

Comment:
@mdeletter: I believe that bug has been fixed; it was unrelated to this code. Please take this out; the tests should still pass without it now.